### DISCUSSION TITLE
2.2.1 Releasse

### DISCUSSION THREAD

**Discussion by Yohn** [Yohn/PicoCSS#21] [Announcements]
# What's Changed
 - Improve RTL support by @remohammadi in #18 
 - Added Popovers - removed Notifications.
 - Added a new Tab component Closes #20 - removed the old tab component (Old tab component did not work in Google Chrome)
    - Valid HTML, 100% accessible, with left and right arrows to navigate the tabs.
    - uses `[role=tablist]` `[role=tab]` and `[role=tabpanel]` attributes.
 - includes pico.fluid.classless.conditional.min.css picocss/pico#645 file missing
 - Improved Dropdowns and Submenus picocss/pico#543 component
 - Corrected dropdown and accordion arrows to function as expected
 - Corrected button colors for light and dark mode
 - Corrected Loading icon in contrast buttons picocss/pico#643 for dark and light mode
 - Safari RTL dropdown picocss/pico#634 text cut off bug corrected
 - Modal: card > form picocss/pico#616 corrected cards to have a form tag before the header and footer and not lose the formatting.
 - Removed margin from last child picocss/pico#557 in forms for better spacing.
 - increased font size for floating labels 

Doc pages updated to include each theme, to easily view how the theme looks in both light and dark modes.

**Comment by sysmat** [comment]
in docs/js remove unused js files

**Reply by Yohn** [reply]
Good point, I'll get that taken care of shortly! Thanks!

**Reply by Yohn** [reply]
I updated the docs directory, to remove the unused files. The npm will have to wait till I do another update which will probably happen within a couple days.
